ush/forecast_det.sh

```diff
@@ -80,9 +84,26 @@ UFS_det(){
       if [[ ! -f "${DATArestart}/CMEPS_RESTART/ufs.cpld.cpl.r.${rdate:0:4}-${rdate:4:2}-${rdate:6:2}-${seconds}.nc" ]]; then
         cmeps_rst_ok="NO"
       fi
-      if [[ ! -f "${DATArestart}/MOM6_RESTART/${rdate:0:8}.${rdate:8:2}0000.MOM.res.nc" ]]; then
       # TODO: add checks for other MOM6 restarts as well
+      if [[ ! -f "${DATArestart}/MOM6_RESTART/${rdate:0:8}.${rdate:8:2}0000.MOM.res.nc" ]]; then
         mom6_rst_ok="NO"
+      else
+        # Also check for MOM6 history file availability
+        # TODO: SFS runs with 24-hr averaging of ocean output, which causes issues with restart checks,
```

Comment:
@aerorahul Can you open an issue for this?

Reply:
https://github.com/NOAA-EMC/global-workflow/issues/4199 has been opened.